modules/ROOT/edits/install-self-managed.adoc

```diff
@@ -0,0 +1,259 @@
+* EU control plane: transport-layer.prod-eu.msap.io | Runtime Fabric message broker for interaction with the control plane.
+| 5044 |TCP (Lumberjack) | * US control plane: dias-ingestor-nginx.prod.cloudhub.io
+* EU control plane: dias-ingestor-nginx.prod-eu.msap.io | Anypoint Monitoring agent for Runtime Fabric.
+| 443 | HTTPS | anypoint.mulesoft.com | Anypoint Platform for pulling assets.
+| 443 | HTTPS | kubernetes-charts.storage.googleapis.com | Kubernetes base charts repository.
+| 443 | HTTPS | docker-images-prod.s3.amazonaws.com | Kubernetes base charts repository.
+| 443 | HTTPS | * US control plane: worker-cloud-helm-prod.s3.amazonaws.com
+* EU control plane: worker-cloud-helm-prod-eu-rt.s3.amazonaws.com worker-cloud-helm-prod-eu-rt.s3.eu-central-1.amazonaws.com | Runtime Fabric version repository. The Runtime Fabric installation uses software from this repository during installation and upgrades.
+| 443 | HTTPS | * US control plane: exchange2-asset-manager-kprod.s3.amazonaws.com
+* EU control plane: exchange2-asset-manager-kprod-eu.s3.amazonaws.com exchange2-asset-manager-kprod-eu.s3.eu-central-1.amazonaws.com |Anypoint Exchange for application assets.
+| 443 | HTTPS | * US control plane: rtf-runtime-registry.kprod.msap.io
+* EU control plane: rtf-runtime-registry.kprod-eu.msap.io | Runtime Fabric Docker repository.
+| 443 | HTTPS | * US control plane: prod-us-east-1-starport-layer-bucket.s3.amazonaws.com prod-us-east-1-starport-layer-bucket.s3.us-east-1.amazonaws.com
+* EU control plane: prod-eu-central-1-starport-layer-bucket.s3.amazonaws.com prod-eu-central-1-starport-layer-bucket.s3.eu-central-1.amazonaws.com | Runtime Fabric Docker image delivery.
+| 443 | HTTPS | * US control plane: runtime-fabric.s3.amazonaws.com
+* EU control plane: runtime-fabric-eu.s3.amazonaws.com | Runtime Fabric Docker repository.
+| 443 | HTTPS | * US control plane: configuration-resolver.prod.cloudhub.io
+* EU control plane: configuration-resolver.prod-eu.msap.io | Anypoint Configuration Resolver.
+|===
+
+=== Certificate Configuration
+
+To allow different endpoints to use mutual TLS authentication to establish a connection, you must configure SSL passthrough to allow the following certificates:
+
+[%header,cols="2*a"]
+.Cerfificate configuration for Self-Managed Kubernetes
+|===
+| Control Plane | Certificates
+| US control plane | transport-layer.prod.cloudhub.io +
+configuration-resolver.prod.cloudhub.io
+| EU control plane | transport-layer.prod-eu.msap.io +
+configuration-resolver.prod-eu.msap.io
+|===
+
+== Step 2: Create a Runtime Fabric using Runtime Manager
+
+[IMPORTANT]
+====
+The procedures in this section should be performed by a MuleSoft organization administrator.
+====
+
+To install Runtime Fabric on Self-Managed Kubernetes, first create a Runtime Fabric using Runtime Manager. This is required to obtain the activation data which is needed during installation.
```

Comment:
ditto previous...and is this, too, a prerequisite to our container topic, which is INSTALLING?
Looks like we need a longer prereq section in this topic OR a separate Level-1 topic Prerequisites to Installing.... or Requirements for Installing....